src/session/dto/getcoupon.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { CouponSchema } from '@/schema/coupons.schema';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CouponDto extends CouponSchema {
+  @ApiProperty({ type: Boolean, description: 'isUseable', required: true })
+  isUseable: boolean;
+
+  constructor(coupon: CouponSchema, isUseable: boolean) {
+    super();
+    Object.assign(this, coupon);
+    this.isUseable = isUseable;
```

Comment:
Since `isUseable` is an API response property, it should be in the snake case.

![image](https://github.com/meeorder/meeorder-backend/assets/56222561/09d72504-5713-4be0-aba7-dadde0e8e1eb)